src/abbreviate.js

```diff
@@ -41,7 +41,7 @@ function parseSuffixes(d, i) {
     @param {Object|String} locale The locale config to be used. If *value* is an object, the function will format the numbers according the object. The object must include `suffixes`, `delimiter` and `currency` properties.
     @returns {String}
 */
-export default function(n, locale = "en-US") {
+export default function(n, locale = "en-US", round = true) {
```

Comment:
This does not disable rounding, but rather abbreviating... which is the whole point of this function! I understand the need for this in d3plus-axis (basically only wanting to use the locale support), but let's have this argument be called `precision`, and is the full string passed to `d3plusFormatLocale.format`. So for example, inside of the `length >= 3` condition, it would say:

```js
const f = formatSuffix(d3plusFormatLocale.format(precision || ".3r")(n), 2, suffixes);
```